packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -14,7 +14,7 @@ import { interaction, selection } from "../state";
  * previously saved applications. Can be supplied an array of filters to use
  * to find files to access instead of the currently selected files.
  */
-export default (filters?: FileFilter[], onDismiss?: () => void) => {
+export default (filters?: FileFilter[], onDismiss?: () => void, isFolder?: boolean) => {
```

Comment:
Is `isFolder` `true` in all the same spots there are non-`undefined` `filters`? I'm wondering if that could be the heuristic and if so should we rename `filters` to be `folderFilters` & remove `isFolder`?

Reply:
refactored in [aceac39](https://github.com/AllenInstitute/biofile-finder/pull/446/commits/aceac39cfceb07295c2cc3aa1358dd1fd161957d)